jpa/spec-tests/src/main/java/ee/jakarta/tck/persistence/se/schemaGeneration/annotations/discriminatorColumn/Client.java

```diff
@@ -131,7 +131,10 @@ public void discriminatorColumnTest() throws Exception {
 		// CREATE TABLE SCHEMAGENSIMPLE (ID INTEGER NOT NULL, PRODUCT_TYPE
 		// VARCHAR(31), PRIMARY KEY (ID))
 
-		pass2 = findDataInFile(f2, "DROP TABLE SCHEMAGENSIMPLE");
+		expected.clear();
+		expected.add("DROP TABLE");
+		expected.add("SCHEMAGENSIMPLE");
```

Comment:
this is likely to change the meaning of the check, for example following file becomes valid:

```
DROP TABLE SOME_TABLE
CREATE TABLE SCHEMAGENSIMPLE
```

given actual check in the `findDataInFile` method is `fileContent.toLowerCase().indexOf(searchString.toLowerCase()) >= 0`

is this intented?

Reply:
AFAIU, `findDataInFile` checks if one line contains all the elements of the list, so the mentioned file would not become valid. One line must contain all the elements of that list.

Reply:
ah, you're right